### PR TITLE
[INTERNAL] feat: add eslint rule to check if insertValue and name are equal

### PR DESCRIPTION
**Pull request by cstrnt**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature

**What is the current behavior? (You can also link to an open issue here)**
Some specs contain equal `insertValue` and `name` props which doesn't have any effect

**What is the new behavior (if this is a feature change)?**
There is a new eslint rule which warns the user about that

**Additional info:**

### CODE REVIEW

**Comment by mattschrage** [comment]
This is a great idea!

File: scripts/eslint-plugin-fig-linter/index.js [hunk 1]
```diff
@@ -5,13 +5,14 @@ module.exports = {
...
     "no-invalid-name": require("./rules/no-invalid-name"),
     "no-empty-array-values": require("./rules/no-empty-array-values"),
+    "no-useless-insertvalue": require("./rules/no-useless-insertvalue"),
   },
   configs: {
     recommended: {
       plugins: ["fig-linter"],
       rules: {
         "fig-linter/no-malicious-script": "error",
-        "fig-linter/no-empty-array-values": "error",
```
Comment (hiradp) [inline scripts/eslint-plugin-fig-linter/index.js:14]: Sorry to chime in, but did you mean to add this rule or change `no-empty-array-values`?

Comment (cstrnt) [reply scripts/eslint-plugin-fig-linter/index.js:14]: Damn, really good find! I wanted to add the rule :D

Comment (QuiiBz) [reply scripts/eslint-plugin-fig-linter/index.js:14]: Hi Tim, could you fix this so this could get merged in? It's a really useful addition!

**Pull request merged**